client/test/test_client.py

```diff
@@ -24,10 +26,23 @@ def test_client(self):
         # Read routes
         self._test_route_return(api_client.get_my_devices(), list)
         self._test_route_return(api_client.get_sites(), list)
-        self._test_route_return(api_client.get_all_alerts(), list)
+        all_alerts = self._test_route_return(api_client.get_all_alerts(), list)
         self._test_route_return(api_client.get_ongoing_alerts(), list)
         self._test_route_return(api_client.get_unacknowledged_alerts(), list)
 
+        # Acknowledge an alert
+        alert_id = None
+        # Try to find an unacknowledged alert
```

Comment:
WOuldn't it be easier to just create one and then acknowledge it ?

Reply:
I was lazy to create device + event (foreign key lock), but I just updated it after your comment!